templates/incl/editor-and-output.html

```diff
@@ -330,3 +319,17 @@
 });
 </script>
 {% endif %}
+
+<script>
+  function getPreviousAndNext(el) {
```

Comment:
Can you take this code to `app.ts` or a similar file where this would conceptually belong? We strive to include as much code as possible inside our TypeScript files! I also like that you didn't use JQuery, but pure JavaScript instead, do you think this is the way moving forward?

Reply:
I think pure JS with the power of hyperscript and htmx is enough currently; until we reach a point we need something more powerful like React. Jquery has far more cons than pros and I really dislike it..